Procfile.dev

```diff
@@ -1,3 +1,3 @@
-web: python3 manage.py runserver_plus
+web: python3 manage.py runserver 0.0.0.0:8000
```

Comment:
Is this change necessary? RunServerPlus is a nice extension for debugging during development.

Reply:
@viadanna, `runserver_plus` doesn't support ASGI. So yes this has to be changed. :(